packages/account/src/Components/address-details/address-details.tsx

```diff
@@ -19,6 +19,7 @@ import { useStatesList } from '@deriv/hooks';
 import { getLocation } from '@deriv/shared';
 import { observer, useStore } from '@deriv/stores';
 import { localize, Localize } from '@deriv/translations';
+import InlineNoteWithIcon from '../inline-note-with-icon';
```

Comment:
can we move this to `InlineMessage` from `components`?

Reply:
I have checked using the component, It has a different design as in `figma` design. We will need to continue to use this `InlineNoteWithIcon` at this moment.

Figma Design:
<img width="980" alt="Screenshot 2023-11-10 at 2 54 56 PM" src="https://github.com/binary-com/deriv-app/assets/103026762/aea264fe-bc82-4926-aa15-8dd25228afde">

Using InlineMessage: 
<img width="1714" alt="Screenshot 2023-11-10 at 2 54 27 PM" src="https://github.com/binary-com/deriv-app/assets/103026762/67dfa745-a1c3-4a75-b756-00d56d9234e0">